iset.mm

```diff
@@ -139302,10 +139302,19 @@ is a function if the structure (without the empty set) is a function.
   baseid $p |- Base = Slot ( Base ` ndx ) $=
     ( cbs c1 df-base 1nn ndxid ) ABCDE $.
 
-  $( Index value of the base set extractor.  (Normally it is preferred to work
-     with ` ( Base `` ndx ) ` rather than the hard-coded ` 1 ` in order to make
-     structure theorems portable.  This is an example of how to obtain it when
-     needed.)  (New usage is discouraged.)  (Contributed by Mario Carneiro,
+  $( Index value of the base set extractor.
+
+     Normally it is preferred to work with ` ( Base `` ndx ) ` rather than the
+     hard-coded ` 1 ` .  For example, using theorems such as ~ baseid and
+     ~ basendxnn .
+
+     The main circumstance in which it is necessary to look at indexes directly
```

Comment:
I don't have a strong opinion on the plural form of index, but consistency is important, and the vast majority of set.mm uses "indices".

Reply:
Heh, I apparently even have less of an opinion than you, so indices it is.